### DISCUSSION TITLE
Skyfield for Simulation of Satellite In-Orbit Validation

### DISCUSSION THREAD

**Discussion by Ly0n** [skyfielders/python-skyfield#1026] [Show and tell]
I just finished my PhD in Atmospheric Physics thanks to Skyfield. You can find the publication here:
[Stray Light Measurement and In-Orbit Validation of an Atmospheric Limb Sounder]( https://elekpub.bib.uni-wuppertal.de/urn/urn:nbn:de:hbz:468-2-5096)

Without Skyfield, I would have needed at least another year.  Thank you all and especially Brandon Rhodes for creating this amazing tool. The code will also be published soon.

**Comment by brandon-rhodes** [comment]
> Without Skyfield, I would have needed at least another year.

WOW!

It is amazing to think that my little astronomy library was able to help you save so much time. Thank you for coming back here to the repository to share this message, it's a great encouragement for me to spend some more time on Skyfield this winter as the snow falls outside. I look forward to reading the materials at your link, and to seeing the code some day if it is indeed released.

**Comment by Ly0n** [comment]
Unfortunately I wasn't able to realise everything I had planned, but I can really see a lot of potential in the combination of Skyfield and [Eradiate](https://github.com/eradiate/eradiate). It would allow you not only to simulate the line-of-sight of a satellite instrument, but also to simulate the observation of that satellite. This is of interest not only for out-of-field stray light simulations, but also for various mission planning simulations for in-orbit observations of stellar objects for validation measurements. 

We have also had great feedback from the satellite mission operators, as Skyfield has allowed us to share basic assumptions in code about when we think a lunar observation is possible.

My successor will now continue my work on the basis of the Skyfield code so that we will be able to validate predicated oberservation with real observation measurements.
